mantle/cmd/kola/testiso.go

```diff
@@ -89,7 +89,7 @@ var (
 		"iso-offline-install.bios",
 		"iso-offline-install.mpath.bios",
 		"iso-offline-install-fromram.4k.uefi",
-		"iso-offline-install-iscsi.ibft.bios",
+		"iso-offline-install-iscsi.ibft.uefi",
```

Comment:
i thought this was just a name ? Just out of curiosity, where is it parsed ?

Reply:
https://github.com/coreos/coreos-assembler/blob/4d4e2840426a6ec70bfd044baca4e4bb2e3fd4fe/mantle/cmd/kola/testiso.go#L584-L588